Meadowlark-js/packages/meadowlark-core/src/security/Security.ts

```diff
@@ -13,12 +13,15 @@ export type Security = {
   authorizationStrategy: AuthorizationStrategy;
   // Client Id string pulled from JWT client_id
   clientId: string;
+  // Whether to validate resources or not, currently pulled from role type 'assessment' from JWT
+  validateResources: boolean;
 };
 
 export function newSecurity(): Security {
   return {
     authorizationStrategy: 'UNDEFINED',
     clientId: 'UNKNOWN',
+    validateResources: true,
```

Comment:
Since we don't have validation in the header, and it needs to be in the FrontendRequest, this felt like the most logical place for validation